### ISSUE TITLE
2.0.1 - Error ... theme could not be found

### ISSUE THREAD

**Issue by dbrosy** [wp-cli/wp-cli#4941]
I have upgraded to 2.0.1 and have an issue with activation of a theme.

running the following command on various versions:

wp theme install <url to theme> --activate

1.2.1 Success
2.0.0 Success
2.0.1 Failure

If I remove the --activate than theme installs
After install I run 
wp theme activate Divi and get the following error:

Error: The 'Divi' theme could not be found

This only happens with 2.0.1

**Comment by jdelvecchio** [comment]
Hi, same error here. Theme cannot be found on activate

**Comment by 16dyui** [comment]
Hi, Theme's name is Internally it is treated in lowercase letters.

```
wp theme install Divin --activate  #can't activate
wp theme install divin --activate  #success
```
Supplement

```
wp theme install Divin
wp theme list
+-----------------+----------+--------+---------+
| name            | status   | update | version |
+-----------------+----------+--------+---------+
| divin           | inactive | none   | 1.1.2   |
+-----------------+----------+--------+---------+
```

**Comment by 16dyui** [comment]
I found Specification change.
[https://github.com/wp-cli/extension-command/pull/126](https://github.com/wp-cli/extension-command/pull/126)

**Comment by 16dyui** [comment]
Issue moved to [wp-cli/extension-command #132](https://github.com/wp-cli/extension-command/issues/132) via [**ZenHub**](https://www.zenhub.com/)

**Comment by schlessera** [comment]
Fixed through #137 with https://github.com/wp-cli/extension-command/releases/tag/v2.0.2
